rust/rsc/src/rsc/add_job.rs

```diff
@@ -69,14 +82,22 @@ pub async fn add_job(
                         mode: Set(out_file.mode),
                         job_id: Set(job_id),
                         blob_id: Set(out_file.blob_id),
-                    });
+                    })
+                    .chunks(65500 / 6)
```

Comment:
arbitrary margin. I'm being slightly cautious in case an update adds more params. I can add a comment when I extract it out. Also at the given scale it should never increase the total number of queries by more than one